src/main/scala/io/iohk/ethereum/network/FastSyncActor.scala

```diff
@@ -0,0 +1,297 @@
+package io.iohk.ethereum.network
+
+import akka.actor.{Actor, ActorLogging, ActorRef, PoisonPill, Props}
+import akka.util.ByteString
+import io.iohk.ethereum.db.storage._
+import io.iohk.ethereum.domain.BlockHeader
+import io.iohk.ethereum.network.FastSyncActor._
+import io.iohk.ethereum.network.PeerActor.MessageReceived
+import io.iohk.ethereum.network.p2p.messages.PV62._
+import io.iohk.ethereum.network.p2p.messages.PV63._
+import io.iohk.ethereum.utils.Config.FastSync
+import org.spongycastle.util.encoders.Hex
+
+class FastSyncActor(
```

Comment:
There is no timeout when sending messages from this actor too peers. I'm not saying in this version we should include that but maybe we need to consider this for future iterations, what do you think @adamsmo ?